pkg/acsengine/const.go

```diff
@@ -87,7 +87,7 @@ var KubeImages = map[string]map[string]string{
 		"dashboard":       "kubernetes-dashboard-amd64:v1.6.1",
 		"exechealthz":     "exechealthz-amd64:1.2",
 		"addonresizer":    "addon-resizer:1.7",
-		"heapster":        "heapster:v1.4.0",
+		"heapster":        "heapster-amd64:v1.4.0",
```

Comment:
These appear to be the same image but I'm happy to keep the naming consistent. Can you please change on all released versions?